usbtool.c

```diff
@@ -27,68 +27,87 @@
 
 enum { INFO, WARNING, ERROR };
 
-void logmsg(int s, char *msg, ...) {
+int isopt(int argc, char **argv, const char *opt){
+
+    for(int i = 1; i < argc; ++i){
+
+        if(strcmp(argv[i], opt) == 0)
```

Comment:
you could move this condition in to the for statement, something like:
```
for(int i = 1; i < argc && strcmp(argv[i], opt) == 0; ++i)
   return 1;
```

Reply:
Nice! :+1: